packages/sdk/src/evm/constants/currency.ts

```diff
@@ -32,6 +32,17 @@ export const NATIVE_TOKENS: Record<number, NativeToken> = {
       symbol: "WETH",
     },
   },
+  // eslint-disable-next-line no-useless-computed-key
+  [11155111]: {
```

Comment:
```suggestion
  11155111: {
```
can't we remove the square brackets so we can remove the eslint disable?

Reply:
@nachoiacovino Done. Btw this pattern still exists on line 188 for the Base Goerli Testnet chain

Reply:
oh cool, can you fix it there too? :D

Reply:
done :)